php/admin/rem_players.php

```diff
@@ -31,7 +32,7 @@
 {
 	$searchresults = array();
 	$playername = $_REQUEST["search_name"];
-	$playername = mysql_real_escape_string($playername); 
+	$playername = mysqli_real_escape_string(SQL_getconnection(), $playername); 
```

Comment:
This would fix the SQL injection in every other place too.

Reply:
Would be at least a minimal better solution, but have to find every query. Would suggest to make it in another change.

Reply:
Sure, this should not be part of this PR.